### PR TITLE
fix duty_id log to be with the correct slot

### PR DESCRIPTION
**Pull request by guym-blox**
### Description 

The log that indicates pruning of old committee runners had the current executed dutyid added to it. This PR replaces it with actual duty id of the pruned runner.

### CODE REVIEW

File: protocol/v2/ssv/validator/committee.go [hunk 1]
```diff
@@ -324,6 +324,7 @@ func (c *Committee) unsafePruneExpiredRunners(logger *zap.Logger, currentSlot ph
 
 	for slot := range c.Runners {
 		if slot <= minValidSlot {
+			logger = logger.With(fields.DutyID(fields.FormatCommitteeDutyID(c.Operator.Committee, c.BeaconNetwork.EstimatedEpochAtSlot(slot), slot)))
```
Comment (y0sher) [inline protocol/v2/ssv/validator/committee.go:327]: did you check this on stage? wouldn't this add 2 duty_id fields since it already has dutyID field?

Comment (guym-blox) [reply protocol/v2/ssv/validator/committee.go:327]: checked with e2e

Comment (y0sher) [reply protocol/v2/ssv/validator/committee.go:327]: and can you confirm my question?

Comment (y0sher) [reply protocol/v2/ssv/validator/committee.go:327]: Did this by passing down a new logger

**Review by y0sher — APPROVED** [review]

**Pull request merged**